### ISSUE TITLE
Make excel file open on row X

### ISSUE THREAD

**Issue by domagojmedo** [ClosedXML/ClosedXML#1656]
**Do you want to request a *feature* or report a *bug*?**
- [ ] Bug
- [ ] Feature
- [x] Question

**Version of ClosedXML**

e.g. 0.95.4

**What is the current behavior?**

We are loading template file and copying it to a new worksheet. After that we fill out the data, add new rows and whatnot. After the file is saved and opened in Excel it won't open on at the top of the sheet, instead it will open at the row that was selected in template .xlsx file before it was saved. 

We tried 
```
  var firstCell = sheet.FirstCell();
  firstCell.Select();
  firstCell.SetActive();
  firstCell.SetValue(' ');
```
right before we save the file, but it doesn't affect the row where the file will be opened. 

Is there an API that can be used to manipulate where the excel file will be opened? Right now we have to make sure that template file was saved with top visible to have it work. 

Attached file will open with row 120 selected. 

[Book1.xlsx](https://github.com/ClosedXML/ClosedXML/files/6335134/Book1.xlsx)

**Comment by Pankraty** [comment]
@domagojmedo
I think this is what you are looking for 
```
sheet.SheetView.TopLeftCellAddress = sheet.FirstCell().Address; // <- Scroll initial view to the top-left
sheet.SelectedRanges.RemoveAll();                               // <- Reset current selection, if any
sheet.FirstCell().SetActive();                                  // <- Make top-left cell selected
```

@igitur, don't you think this may worth a helper method, e.g. `IXLWorksheet.ResetViewToDefault()`?

**Comment by domagojmedo** [comment]
@Pankraty if helper method will be added, maybe it would be more helpful to be able to set default selection?

**Comment by Pankraty** [comment]
Agree. Also, it should reset zoom factors to defaults and change `XLSheetView.View` to `Normal`.

**Comment by domagojmedo** [comment]
@Pankraty I just got around to testing it and it does work, thanks
